internal/output/singular_outputter.go

```diff
@@ -0,0 +1,34 @@
+package output
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+var singularPlaintextOutputFn = func(r resource) string {
```

Comment:
We're going to end up with a few of these functions depending on the shape of the response.